CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## 0.4.8
+
+### Performance
+
+- Maintain synchronicity between the lexer and the parser ([#11457](https://github.com/astral-sh/ruff/pull/11457))
```

Comment:
I would lead with the user-visible change, and then explain how it was achieved:

```suggestion
- Linter performance has been improved by 25% or more on some microbenchmarks
  by refactoring the lexer and parser to maintain synchronicity between them
  ([#11457](https://github.com/astral-sh/ruff/pull/11457))
```

Reply:
I don't think it's 25% xD

It's the lexer that's improved to around 25% while the linter is at around 10%

Reply:
But, I agree with this wording, thanks for the suggestion

Reply:
Ah right, yeah :)

("Only" 10% 😆)